### PR TITLE
Stop updating RUBY_VERSION and BUNDLED_WITH details in Ruby lockfiles

### PR DESCRIPTION
**Pull request by greysteil**
Previously, we were nobbling both. Now we won't nobble either.

### CODE REVIEW

File: lib/bump/dependency_file_updaters/ruby.rb [hunk 1]
```diff
@@ -76,28 +78,25 @@ def updated_gemfile_lock_content
...
+              definition.to_lock
+            end
           end
-        end
+        post_process_lockfile(lockfile_body)
       rescue SharedHelpers::ChildProcessFailed => error
-        if error.error_class == "Bundler::VersionConflict"
-          raise DependencyFileUpdaters::VersionConflict
-        end
+        raise unless error.error_class == "Bundler::VersionConflict"
```
Comment (greysteil) [inline lib/bump/dependency_file_updaters/ruby.rb:97]: Unrelated style cleanup.

File: spec/dependency_file_updaters/ruby_spec.rb [hunk 2]
```diff
@@ -116,14 +114,33 @@
...
+        expect(file.content).to include "RUBY VERSION\n   ruby 2.2.0p0"
+      end
+    end
+
+    context "when the Gemfile.lock didn't have a BUNDLED WITH line" do
+      let(:lockfile_body) do
+        fixture("ruby", "lockfiles", "no_bundled_with.lock")
+      end
+
+      it "doesn't add in a BUNDLED WITH" do
```
Comment (hmarr) [inline spec/dependency_file_updaters/ruby_spec.rb:145]: Add equivalent spec for RUBY VERSION

**Review by hmarr — APPROVED** [review]
👍  when additional spec is added

File: spec/dependency_file_updaters/ruby_spec.rb (hunk 2 above)
Comment (greysteil) [reply spec/dependency_file_updaters/ruby_spec.rb:145]: Done.
